syntax-sloc/lang-file-sloc.rkt

```diff
@@ -6,20 +6,36 @@
 
 (provide lang-file-sloc)
 
-(require "syntax-sloc.rkt"
-         typed/syntax-sloc/read-lang-file
-         syntax-sloc/syntax-sloc)
+(require racket/set
+         syntax-sloc/syntax-sloc
+         typed/syntax-sloc/read-lang-file)
 
 (module+ test
   (require typed/rackunit
            racket/runtime-path))
 
-(: lang-file-sloc : Path-String [#:use-stx? Stx-Pred] -> Natural)
-(define (lang-file-sloc path-string #:use-stx? [use-stx? #f])
-  (syntax-sloc (read-lang-file path-string) #:use-stx? use-stx?))
+(: lang-file-sloc : Path-String [#:include-stx? Stx-Pred] -> Natural)
+(define (lang-file-sloc path-string #:include-stx? [include-stx? #f])
```

Comment:
Habit. I like keywords to take `#f` even when the right value is simple to make it easy for higher-order functions or macros to supply a default argument.